test/run_test.py

```diff
@@ -1218,6 +1220,18 @@ def parse_args():
         help="Runs the full test suite despite one of the tests failing",
         default=strtobool(os.environ.get("CONTINUE_THROUGH_ERROR", "False")),
     )
+    parser.add_argument(
+        "--pipe-logs",
+        action="store_true",
+        help="Print logs to output file while running tests.  True if in CI and env var is not set",
+        default=IS_CI and not strtobool(os.environ.get("VERBOSE_TEST_LOGS", "False")),
+    )
+    parser.add_argument(
+        "--set-timeout",
```

Comment:
This option looks a bit weird as a bool.  Its name and description points to a timeout value instead, i.e. `--set-timeout 0` to disable timeout or `--set-timeout 3600`.  May be choose another name like `--enable-timeout` would make this clearer.

Reply:
swapped to enable timeout